src/evm_parse.c

```diff
@@ -189,7 +201,66 @@ uint256_t enforceParsedScalarFits256Bits(struct EVM_RLP_item_state *const state)
   return value;
 }
 
-enum parse_rv parse_rlp_txn(struct EVM_RLP_list_state *const state, evm_parser_meta_state_t *const meta) {
+IMPL_FIXED(uint8_t)
```

Comment:
Isn't this duplicate of the one added in 
https://github.com/obsidiansystems/ledger-app-avalanche/blob/b1612f096aa61c48f7b86a63821425b3f823e7e0/src/parser.c#L111

Reply:
It is. Should `IMPL_FIXED(uint8_t);` therefore be moved to `parser.h` so both `parser.c` and `evm_parse.c` can import it?

Reply:
Yop